recipes/qt_binder/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2" %}
+
+package:
+  name: qt_binder
+  version: {{ version }}
+
+source:
+  fn: qt_binder-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/q/qt_binder/qt_binder-{{ version }}.tar.gz
+  sha256: 392a105a6a1734e1f67e17b88ee72225be594dfc0000dc3d53e254a1b8a532e5
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - libpng >=1.6.22  # current qt 4.8.7 3 from defaults requires libpng 1.6.22
```

Comment:
@ocefpaf, @jakirkham
`qt` from defaults seems to work fine with the libpng 1.6.23 that is now on conda-forge.  I am not sure what the best solution here is since there is no guarantee that the `qt` from defaults doesn't get out of sync with the `libpng` here.  What do you recommend in terms of the pinning?